### PR TITLE
FORTH: update to generate InvalidOperationException from canonical data

### PR DESCRIPTION
**Pull request by bjorkqvist**
As per instructed in #648 :

Regenerated test class.
Changed generator to generate same testfile.
Tests passed.

### CODE REVIEW

File: generators/Exercises/Generators/Forth.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ protected override void UpdateTestMethod(TestMethod testMethod)
         {
             testMethod.TestMethodName = testMethod.TestMethodNameWithPath;
 
-            if (testMethod.Expected == null)
+            if (testMethod.Expected == null || string.Join(" ", testMethod.Expected).StartsWith("[error"))
```
Comment (ErikSchierboom) [inline generators/Exercises/Generators/Forth.cs]: Is the `null` check still necessary? I think you could replace the if-expression with `testMethod.Expected is Dictionary<string, object>`, which is how other generators also do this check.

**Comment by bjorkqvist** [comment]
Sure, i will give it a try. 😊

Den tors 4 okt. 2018 08:05Erik Schierboom <notifications@github.com> skrev:

**Comment by bjorkqvist** [comment]
ok, so Emazculator is an old account of mine. Don't understand  why it got the latest commit. The commit should be fine, but i'm deleting my old account to avoid future confusion.

**Comment by ErikSchierboom** [comment]
Merged. Thanks a lot! 🎉

**Pull request merged**